java/memory/src/main/java/io/netty/buffer/UnsafeDirectLittleEndian.java

```diff
@@ -268,4 +267,19 @@ public int hashCode() {
     return System.identityHashCode(this);
   }
 
+  @Override
+  public boolean equals(Object obj) {
+    if (this == obj) {
+      return true;
+    }
+
+    if (obj instanceof UnsafeDirectLittleEndian) {
+      UnsafeDirectLittleEndian that = (UnsafeDirectLittleEndian) obj;
+      return this.wrapped.equals(that.wrapped) &&
```

Comment:
This does not seem to be consistent with `hashCode`, which is based on object identity. To be consistent, `equals` should just return `this == obj`.

Reply:
Good point, thanks